cmake_modules/FindANTLR.cmake

```diff
@@ -0,0 +1,118 @@
+################################################################################
+# Copyright (C) 2013 HPCC Systems.
+#
+# All rights reserved. This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+################################################################################
+# - Attempt to find the ANTLR jar needed to build and run ANTLR Lexers and Parsers
+# Once done this will define
+#
+# ANTLR_FOUND - ANTLR found in local system
+# ANTLR_BUILDTIME_JAR - The jar needed to build/generate ANTLR Lexers and Parsers
+# ANTLR_RUNTIME_JAR - The jar needed to build/generate ANTLR Lexers and Parsers
+################################################################################
+
+include(UseJava)
+
+option(ANTLR_VER "ANTLR runtime and buildtime version.")
+if(NOT ANTLR_VER)
+    set(ANTLR_VER 3.4)
+endif()
+
+option(ANTLR_BUILDTIME_DEP "ANTLR buildtime jar file name.")
+if(NOT ANTLR_BUILDTIME_DEP)
+    set(ANTLR_BUILDTIME_DEP "antlr-${ANTLR_VER}-complete")
+    message(STATUS "Option ANTLR_BUILDTIME_DEP not set, setting to: ${ANTLR_BUILDTIME_DEP}")
+    message(STATUS "The jar can be downloaded directly from:\n\thttps://github.com/antlr/website-antlr3/raw/gh-pages/download/antlr-3.4-complete.jar")
```

Comment:
let's automatically download these... can that be a submodule? 
also, let's make a USE_ANTLR option, if false, don't build Wssql and proceed w/ the build